_templates/header.php

```diff
@@ -71,32 +76,18 @@
         <link rel="stylesheet" type="text/css" media="all" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,300italic,400italic|Droid+Sans|Roboto+Mono">
         <?php } ?>
 
-        <link rel="stylesheet" type="text/css" media="all" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+        <link rel="stylesheet" type="text/css" media="all" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.5.0/css/font-awesome.min.css">
         <link rel="stylesheet" type="text/css" media="all" href="styles/main.css">
 
-        <script>
-            (function(d,s,f){g=d.createElement(s),u=d.getElementsByTagName(s)[0],g.async=1,g.src=f,u.parentNode.insertBefore(g,u)})
-            (document,'script','https:<?php echo $sitewide['branch_root']; ?>backend/hsts.php')
-        </script>
-
-        <script>
-            <?php include __DIR__.'/../scripts/jql.min.js'; ?>
-            jQl.loadjQ('//cdn.jsdelivr.net/g/jquery')
-            jQl.boot()
-            <?php include __DIR__.'/../scripts/popover.js'; ?>
-            <?php include __DIR__.'/../scripts/smooth-scrolling.js'; ?>
-            <?php include __DIR__.'/../scripts/twitter-links.js'; ?>
-            <?php include __DIR__.'/../scripts/external-links.js'; ?>
-        </script>
-
-        <?php echo !empty($page['scripts']) ? $page['scripts'] : false; ?>
+        <?php foreach ($page['styles'] as $style) { ?>
+        <link rel="stylesheet" type="text/css" media="all" href="<?php echo $style ?>">
+        <?php } ?>
 
+        <script async src="https:<?php echo $sitewide['branch_root'] ?>backend/hsts.php"></script>
```

Comment:
I can confirm this is not a problem with midori on 16.04

![screenshot from 2016-08-20 09 56 50](https://cloud.githubusercontent.com/assets/3385679/17832632/7f8db28a-66bc-11e6-8c89-9a095e2bfec8.png)